Releasing
=========

:Contact: grubert@users.sourceforge.net, docutils-develop@users.sourceforge.net
:Revision: $Revision: 7396 $
:Date: $Date: 2012-04-29 11:18:23 +0200 (So, 29 Apr 2012) $
:Copyright: This document has been placed in the public domain.

Notes on what happend while releasing.

Release 0.9.1
=============

same failures as for 0.9 plus

* python 2.3: twice, ignored ::

  -             [Errno 2] No such file or directory: 'bogus.csv'.
  +             [Errno 2] No such file or directory: u'bogus.csv'.


Release 0.9
===========

* python 2.3 unittest.TestCase has no assertTrue
* python 2.3 keyword dictionaries update method does not support kwargs

Ignored test errors

* python 2.3: unicode problems. 2.3 support is likely to be ended soon.
  Systems with only 2.3 might not even know of unicode.

* PIL ``AttributeError: 'module' object has no attribute 'Image'``

  A problem in PIL ? 
  Tested and failiing on

  - ubuntu8.04, python 2.4, 2.5  
  - ubuntu10.04, python 2.6  
  - ubuntu11.10, python 2.7  

* python 2.64rc1 has a different error message format::

              Problems with "raw" directive path:
       -      InputError: [Errno 2] No such file or directory: 'non-existent.file'.
       +      InputError: (2, 'No such file or directory').

  but not in python 2.6.7
